lego/apps/users/models.py

```diff
@@ -445,8 +448,10 @@ def number_of_penalties(self) -> int:
         # Returns the total penalty weight for this user
         count = (
             Penalty.objects.valid()
-            .filter(user=self)
-            .aggregate(models.Sum("weight"))["weight__sum"]
+            .filter(
+                penalty_group__user=self,
+            )
+            .count()
```

Comment:
This does not really make that much sense anymore, as a user never has any more than one penalty, but I suppose it doesn't hurt